py/desisim/scripts/quickbrick.py

```diff
@@ -120,9 +120,18 @@ def main(args):
     log.debug('Using OBJTYPE {}'.format(objtype))
     if objtype == 'BGS' or objtype == 'MWS' or objtype == 'BRIGHT_MIX':
         qsim.observation.exposure_time = desiparams['exptime_bright'] * u.s
-        qsim.atmosphere.moon.moon_zenith = args.moon_zenith * u.deg
-        qsim.atmosphere.moon.separation_angle = args.moon_angle * u.deg
-        qsim.atmosphere.moon.moon_phase = args.moon_phase
+        if args.moon-phase is None:
+            qsim.atmosphere.moon.moon_phase = 0.7
+        else:
+            qsim.atmosphere.moon.moon_phase = args.moon-phase
+        if args.moon-angle is None:
+            qsim.atmosphere.moon.separation_angle = 50 * u.deg
+        else:
+            qsim.atmosphere.moon.separation_angle = args.moon-angle * u.deg
+        if args.moon-zenith is None:
+            qsim.atmosphere.moon.moon_zenith = 30 * u.deg
+        else:
+            qsim.atmosphere.moon.moon_zenith = args.moon-zenith * u.deg
     else:
         qsim.observation.exposure_time = desiparams['exptime_dark'] * u.s
 
```

Comment:
In quickgen you also have cases for gray/grey, which don't appear here.  Please add.